downstream/modules/platform/con-cluster-platform-ext-database.adoc

```diff
@@ -9,7 +9,7 @@ This scenario includes installation of multiple {ControllerName} nodes and an {H
 [NOTE]
 ====
 * Running in a cluster setup requires any database that {ControllerName} uses to be external--PostgreSQL must be installed on a machine that is not one of the primary or secondary tower nodes. When in a redundant setup, the remote PostgreSQL version requirements is *PostgreSQL 12*.
-** See link:https://docs.ansible.com/ansible-tower/3.8.2/html/administration/clustering.html#ag-clustering[Clustering] for more information on configuring a clustered setup.
+** See link:https://docs.ansible.com/automation-controller/latest/html/quickstart/index.html#ag-clustering[Clustering] for more information on configuring a clustered setup.
```

Comment:
I couldn't find a Clustering section in the Quickstart Guide - or is there a plan to move the section from the admin guide to the quickstart guide?

I found the clustering section in the controller guide:
https://docs.ansible.com/automation-controller/latest/html/administration/clustering.html

Reply:
I will have to ask Chris. These changes are based upon the ticket that had been opened up. [AAP-1527](https://issues.redhat.com/browse/AAP-1257). From our conversation on chat, and the list in the ticket, it appeared to belong to the same URL. We shall see what Chris has to say.